Assets/MixedRealityToolkit/_Core/Extensions/CollectionsExtensions.cs

```diff
@@ -135,11 +136,11 @@ public static T[] ExportDictionaryValuesAsArray<T>(this Dictionary<uint,T> input
         /// <param name="key">The specific DeviceInputType value to update</param>
         /// <param name="value">The data value to be updated</param>
         public static void SetDictionaryValue<T>(
-            this Dictionary<Definitions.Devices.DeviceInputType, Definitions.Devices.InteractionMapping> input, 
+            this Dictionary<Definitions.Devices.DeviceInputType, Interfaces.IInteractionMapping> input, 
             Definitions.Devices.DeviceInputType key, T value)
         {
-            var entry = input[key];
-            entry.SetValue(value);
+            var entry = input[key] as InteractionMapping<T>;
+            entry.GetValue(value);
```

Comment:
Should we check if the entry is not null?

Reply:
Quite possibly, worth adding